includes/ui/Settings.php

```diff
@@ -0,0 +1,268 @@
+</svg>
+',
+			]
+		);
+
+		$elements[] = Notice::render(
+			[
+				'title'   => 'Test Notice Created',
+				'icon'    => 'check',
+				'message' => 'This is a test Job Manager notice. Should be large with a checkmark.',
+				'buttons' => [
+					[
+						'text' => 'Primary Button',
+						'href' => '/',
+					],
+					[
+						'text' => 'Secondary Button',
+						'href' => '/',
+					],
+				],
+				'links'   => [
+					[
+						'text' => 'Action',
+						'href' => '/',
+					],
+					[
+						'text' => 'Action',
+						'href' => '/',
+					],
+				],
+			]
+		);
+
+		$elements[] = Notice::render(
+			[
+				'title'   => 'Test Notice Created',
+				'message' => 'This is a test Job Manager notice. Should be large with a checkmark.',
+				'buttons' => [
+					[
+						'text'    => 'Light Button',
+						'href'    => '/',
+						'primary' => false,
+					],
+				],
+				'links'   => [
+					[
+						'text' => 'Action',
+						'href' => '/',
+					],
+				],
+			]
+		);
+
+		$elements[] = Notice::render(
+			[
+				'title'   => 'Test Notice Created',
+				'message' => 'This is a test Job Manager notice.',
+				'links'   => [
+					[
+						'text' => 'Action',
+						'href' => '/',
+					],
+					[
+						'text' => 'Action',
+						'href' => '/',
+					],
+				],
+			]
+		);
+
+		$elements[] = Notice::render(
+			[
+				'title'   => 'Test Notice Created',
+				'icon'    => 'check',
+				'message' => 'This is a test Job Manager notice. Should be large with a checkmark.',
+				'buttons' => [
+					[
+						'text' => 'Primary Button',
+						'href' => '/',
+					],
+				],
+			]
+		);
+
+		$elements[] = Notice::render(
+			[
+				'icon'    => 'check',
+				'message' => 'This is a test Job Manager notice, with an icon, message and an action.',
+				'links'   => [
+					[
+						'text' => 'Action',
+						'href' => '/',
+					],
+				],
+			]
+		);
+
+		$elements[] = Notice::render(
+			[
+				'icon'    => 'check',
+				'classes' => [ 'alignwide' ],
+				'message' => 'This is a wide Job Manager notice, with an icon, message and an action. Actions are on the side.',
+				'links'   => [
+					[
+						'text' => 'Action',
+						'href' => '/',
+					],
+				],
+			]
+		);
+
+		$elements[] = Notice::render(
+			[
+				'message' => 'This is a test Job Manager notice, with only a message and an action.',
+				'links'   => [
+					[
+						'text' => 'Action',
+						'href' => '/',
+					],
+				],
+			]
+		);
+
+		$elements[] = Notice::render(
+			[
+				'message' => 'This is a test Job Manager notice.',
+				'buttons' => [
+					[
+						'text' => 'Primary Button',
+						'href' => '/',
+					],
+				],
+			]
+		);
+
+		$elements[] = Notice::render(
+			[
+				'message' => 'This is a test Job Manager notice.',
+				'buttons' => [
+					[
+						'text'    => 'Outlined Button',
+						'href'    => '/',
+						'primary' => false,
+					],
+				],
+			]
+		);
+
+		return implode( '', $elements );
```

Comment:
Instead of using an intermediary array, can you just use a string and append ?
$elements .= Notice::render(blablabla)

Reply:
Kept this one as array, just to allow tweaking the rendering later, like adding some separator between the items.